Makefile.js

```diff
@@ -143,6 +143,20 @@ function generateBlogPost(releaseInfo) {
 }
 
 /**
+ * Generates a doc page with formatter result examples
+ * @param  {Object} formatterInfo Linting results from each formatter
+ * @returns {void}
+ */
+function generateFormatterExamples(formatterInfo) {
+    var output = ejs.render(cat("./templates/formatter-examples.md.ejs"), formatterInfo),
+        filename = "../eslint.github.io/docs/user-guide/formatters/index.md",
+        htmlFilename = "../eslint.github.io/docs/user-guide/formatters/html-formatter-example.html";
+
+    output.to(filename);
+    fs.writeFileSync(htmlFilename, formatterInfo.formatterResults.html.result);
```

Comment:
Thanks, I wasn't trying to push back and will definitely change it. I just
haven't had an opportunity, and was curious where it came from. Sorry to
say I copied your blog creation code without really understanding where
`to` came from. :(
On Mon, Sep 14, 2015 at 7:17 PM Nicholas C. Zakas <notifications@github.com>
wrote:

> In Makefile.js
> <https://github.com/eslint/eslint/pull/3741#discussion_r39459125>:
>
> > @@ -143,6 +143,20 @@ function generateBlogPost(releaseInfo) {
> >  }
> >
> >  /**
> > + * Generates a doc page with formatter result examples
> > + * @param  {Object} formatterInfo Linting results from each formatter
> > + * @returns {void}
> > + */
> > +function generateFormatterExamples(formatterInfo) {
> > +    var output = ejs.render(cat("./templates/formatter-examples.md.ejs"), formatterInfo),
> > +        filename = "../eslint.github.io/docs/user-guide/formatters/index.md",
> > +        htmlFilename = "../eslint.github.io/docs/user-guide/formatters/html-formatter-example.html";
> > +
> > +    output.to(filename);
> > +    fs.writeFileSync(htmlFilename, formatterInfo.formatterResults.html.result);
>
> Yes, it's coming from ShellJS. And you're using it on the immediately
> preceding line (155). :) We should be consistent throughout.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eslint/eslint/pull/3741/files#r39459125>.
>